UnityProject/Assets/Scripts/Furniture/OccupiableDirectionalSprite.cs

```diff
@@ -170,7 +187,7 @@ private void SyncOccupant(uint occupantNewValue)
 	//ensures we are rendering in the correct sprite layer
 	private void EnsureSpriteLayer()
 	{
-		if (directional.CurrentDirection == Orientation.Up && occupant > 0)
+		if (directional.CurrentDirection == Orientation.Up && occupantNetId > 0)
```

Comment:
Will be preferable to use the NetId.Empty constant instead of 0 (makes it more pretty)